dlt/pipeline/helpers.py

```diff
@@ -152,13 +125,14 @@ def is_empty(self) -> bool:
             and len(self.info["resource_states"]) == 0
         )
 
-    def _drop_destination_tables(self) -> None:
-        table_names = [tbl["name"] for tbl in self.tables_to_drop]
-        for table_name in table_names:
-            assert table_name not in self.schema._schema_tables, (
-                f"You are dropping table {table_name} in {self.schema.name} but it is still present"
-                " in the schema"
-            )
+    def _drop_destination_tables(self, allow_schema_tables: bool = False) -> None:
```

Comment:
Do we still need it? we load state package anyway so all this modifications happen in the load right?

Reply:
Nope, now drop command creates a load package: https://github.com/dlt-hub/dlt/blob/sthor%2Ffull-refresh-adjustment/dlt/pipeline/helpers.py#L150-L154